Resources/Prototypes/Entities/Objects/Specific/chemistry.yml

```diff
@@ -17,7 +17,6 @@
   - type: Pourable
     transferAmount: 5.0
   - type: Spillable
-  - type: GlassBeaker
```

Comment:
![grafik](https://user-images.githubusercontent.com/16618384/101892591-e0abde00-3ba3-11eb-8b27-6a73fd24e114.png)

Reply:
can you please mark this as a suggested change again? It doesn't give me an option to commit the suggestion for some reason.